### PR TITLE
TESTING PR

### PR DESCRIPTION
**Pull request by maxklema**

### CODE REVIEW

**Review by horner — CHANGES_REQUESTED** [review]
Let’s add the tests and an action to run them

File: readme.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ A package designed to interact with [MIE's](https://www.mieweb.com/) (Medical In
 * [Motivation](#Motivation)
 * [Author](#Author)
 
+## Testing
```
Comment (horner) [inline readme.md:28]: This is an inline comment

**Comment by maxklema** [comment]
Reopen branch.

**Comment by maxklema** [comment]
@horner This PR has been moved here: https://github.com/maxklema/mie-api-tools/pull/20

**Pull request closed without merging**